dev/integration_tests/android_testing/lib/src/matcher.dart

```diff
@@ -0,0 +1,177 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:test/test.dart';
+
+import 'common.dart';
+import 'constants.dart';
+
+/// Matches an [AndroidSemanticsNode].
+///
+/// Any properties which aren't supplied are ignored during the comparison.
+///
+/// This matcher is intended to compare the accessibility values generated by
+/// the Android accessibility bridge, and not the semantics object created by
+/// the Flutter framework.
```

Comment:
Maybe mention that unspecified/null values are not matched against

Reply:
See previous sentence

Reply:
woops